CIP-0144/README.md

```diff
@@ -0,0 +1,1604 @@
+  "operation": "getCommitteeMemberById",
+  "request": {
+    "$ref": "#/cip-139/CCHotId"
+  },
+  "response": {
+    "$ref": "#/cip-139/CCMember"
+  },
+  "errors": [
+    {
+      "$ref": "#/appendix/APIError"
+    }
+  ]
+}
+```
+
+Get a specific Committee member by id.
+
+##### Pool
+
+###### All
+
+```
+{
+  "operation": "getAllPools",
+  "request": {},
+  "response": {
+    "type": "object",
+    "properties": {
+      "pools": {
+        "type": "array",
+        "items": {
+          "$ref": "#/cip-139/Pool"
+        }
+      }
+    },
+    "required": [
+      "pools"
+    ]
+  },
+  "errors": [
+    {
+      "$ref": "#/appendix/APIError"
+    }
+  ]
+}
+```
+
+Get all known stake pools.
+
+###### Id
+
+```
+{
+  "operation": "getPoolById",
+  "request": {
+    "$ref": "#/cip-139/PoolPubKeyHash"
+  },
+  "response": {
+    "$ref": "#/cip-139/Pool"
+  },
+  "errors": [
+    {
+      "$ref": "#/appendix/APIError"
+    }
+  ]
+}
+```
+
+Get a specific stake pool by id.
+
+##### Proposal
+
+###### All
+
+```
+{
+  "operation": "getAllProposals",
+  "request": {},
+  "response": {
+    "type": "object",
+    "properties": {
+      "proposals": {
+        "type": "array",
+        "items": {
+          "$ref": "#/cip-139/Proposal"
+        }
+      }
+    },
+    "required": [
+      "proposals"
+    ]
+  },
+  "errors": [
+    {
+      "$ref": "#/appendix/APIError"
+    }
+  ]
+}
+```
+
+Get all known proposals.
+
+###### Id
+
+```
+{
+  "operation": "getProposalById",
+  "request": {
+    "$ref": "#/cip-139/ProposalId"
+  },
+  "response": {
+    "$ref": "#/cip-139/Proposal"
+  },
+  "errors": [
+    {
+      "$ref": "#/appendix/APIError"
+    }
+  ]
+}
+```
+
+Get a specific proposal by id.
+
+##### Era
+
+###### Summary
+
+```
+{
+  "operation": "getEraBySummary",
+  "request": {},
+  "response": {
+    "type": "object",
+    "properties": {
+      "summary": {
+        "type": "array",
+        "items": {
+          "$ref": "#/cip-139/EraSummary"
+        }
+      }
+    },
+    "required": [
+      "summary"
+    ]
+  },
+  "errors": [
+    {
+      "$ref": "#/appendix/APIError"
+    }
+  ]
+}
+```
+
+Get the start and end of each era along with parameters that can vary between hard forks.
+
+
+### Versioning
+
+In this CIP we are defining two different APIs: the connection API for wallets, and the CIP-30 *[maybe this should have another name to prevent confusion?]* extension which enables an own-data wallet. These two are separate components, below there is a table with separate entries for the versions of the connection API and CIP-30 Extension respectively. 
+
+While the CIP is in preparation, these versions shall be set to `0.0.0`. The moment this CIP is merged the versions shall be set to `1.0.0`, and all implementations should consider that the current version. Any changes to either API should come in form of PRs to this CIP.
+
+|     |    |
+| --- | --- |
+| Version-Connection-API | 0.0.0 |
+| Version-CIP-30-Extension | 0.0.0 |
```

Comment:
```suggestion
| API | Version |
| --- | --- |
| Version-Connection-API | 0.0.0 |
| Version-CIP-30-Extension | 0.0.0 |
```
... follow up to https://github.com/cardano-foundation/CIPs/pull/957#discussion_r1908650806 & https://github.com/cardano-foundation/CIPs/pull/957/commits/1b4e5b36837d9f67a3cea8052962ea52319eaf69, to either give column headers to this table... or the header bar could be removed by cutting the first two lines.